packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -85,6 +94,8 @@ interface BodyProps {
     url: string;
     menu?: React.ReactNode;
     appMenu?: React.ReactNode;
+    // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
would be nice to provide some explanation, for example link to the issue

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unused-prop-types.md#false-positives-sfc